assets/js/validator.js

```diff
@@ -0,0 +1,30 @@
+const nameField = document.getElementById('fullname');
+const emailField = document.getElementById('email-field');
+const messageField = document.getElementById('message-field');
+const submitButton = document.getElementById('submit');
+
+const nameErrorMsg = document.getElementById('name-error');
+const emailErrorMsg = document.getElementById('email-error');
+const messageError = document.getElementById('message-error');
+
+submitButton.addEventListener('click', () => {
+  if (nameField.value.length === 0) {
+    nameErrorMsg.innerHTML = 'Your Name is required';
+  } else {
+    nameErrorMsg.innerHTML = '';
+  }
+
+  if (emailField.value.length === 0) {
+    emailErrorMsg.innerHTML = 'Your Email is required';
+  } else if (emailField.value !== emailField.value.toLowerCase()) {
+    emailErrorMsg.innerHTML = 'Please input your email address in lowercase.';
+  } else {
+    emailErrorMsg.innerHTML = '';
```

Comment:
- [ ] Well done checking if the email is all lowercase by comparing if the email and it's lowercase converted version are the same using` email === email.toLowerCase()`, however when I tried it with an email with Uppercase, I've got the error message as required but the form was submitted normally. Kindly try to fix the issue, if the email is in Uppercase the form should not be submitted.

Reply:
Well resolved 👍🏽